electron/legendary/games.ts

```diff
@@ -243,6 +243,28 @@ class LegendaryGame extends Game {
     return icon
   }
 
+  private shortcutFiles(gameTitle: string) {
+    let desktopFile
+    let menuFile
+
+    switch (process.platform) {
+    case 'linux': {
+      desktopFile = `${app.getPath('desktop')}/${gameTitle}.desktop`
+      menuFile = `${home}/.local/share/applications/${gameTitle}.desktop`
+      break
+    }
+    case 'win32': {
+      desktopFile = `${app.getPath('desktop')}\\${gameTitle}.lnk`
+      menuFile = `${app.getPath('appData')}\\Microsoft\\Windows\\Start Menu\\Programs\\${gameTitle}.lnk`
```

Comment:
Even though in spanish the Start menu is `Menú Inicio`, both `Start Menu` and `Menú Inicio` work, Windows understands `Start Menu` and `Programs` independent of the current system language